gradio/components/number.py

```diff
@@ -121,12 +121,14 @@ def preprocess(self, payload: float | None) -> float | int | None:
         """
         if payload is None:
             return None
-        elif self.minimum is not None and payload < self.minimum:
+
+        if self.minimum is not None and payload < self.minimum:
             raise Error(f"Value {payload} is less than minimum value {self.minimum}.")
         elif self.maximum is not None and payload > self.maximum:
             raise Error(
                 f"Value {payload} is greater than maximum value {self.maximum}."
             )
+
```

Comment:
IIUC this change should have no effect since the first branch terminates in a return statement, am I misunderstanding?